### DISCUSSION TITLE
weird sprite scaling inconsistency

### DISCUSSION THREAD

**Discussion by Tripodfish-the-sequal** [FakeFishGames/Barotrauma#14666] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

For some reason, the exosuits changes the scale of its arms depending if it's on the ground or being worn. This problem is just visual but has been annoying me for some time. The arms are much smaller when worn making the exosuit look like a massive box with shrimpy arms

### Reproduction steps

Place one exosuit on ground
Wear one exosuit
Cry for hours

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, fix coming up in a future update!
